<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color:rgb(33, 33, 41);
font-family: 'Sans-Serif';
 font-size: 20px;    
 background-color: rgb(209, 255, 117);;
border-radius:20px;
font-color:white;


</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>354</x>
       <y>220</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connection</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_32">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#d1d938;&quot;&gt;employees&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color:rgb(33, 33, 41);
font-family: 'Sans-Serif';
 font-size: 24px;    
 background-color: transparent;
border-radius:20px;
font-color:white;


}</string>
     </property>
     <property name="text">
      <string>CIN:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    font-family: 'Sans-Serif';
    font-size: 20px;
    background-color: white;  /* Couleur de fond blanche */
    border-radius: 20px;
}
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Normal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_37">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#d1d938;&quot;&gt;employees&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color:rgb(33, 33, 41);
font-family: 'Sans-Serif';
 font-size: 24px;    
 background-color: transparent;
border-radius:20px;
font-color:white;


}</string>
     </property>
     <property name="text">
      <string>PASSWORD</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cin">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    font-family: 'Sans-Serif';
    font-size: 20px;
    background-color: white;  /* Couleur de fond blanche */
    border-radius: 20px;
}
</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>150</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Mot depasse oublier</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>541</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
color:rgb(33, 33, 41);
font-family: 'Sans-Serif';
 font-size: 20px;    
 background-color: rgb(209, 255, 117);;
border-radius:20px;
font-color:white;


</string>
     </property>
     <widget class="QPushButton" name="pushButton_33">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#d1d938;&quot;&gt;employees&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color:rgb(33, 33, 41);
font-family: 'Sans-Serif';
 font-size: 24px;    
 background-color: transparent;
border-radius:20px;
font-color:white;


}</string>
      </property>
      <property name="text">
       <string>CIN :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cin_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    font-family: 'Sans-Serif';
    font-size: 20px;
    background-color: white;  /* Couleur de fond blanche */
    border-radius: 20px;
}
</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>enovyer un code de recupuration</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>541</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
color:rgb(33, 33, 41);
font-family: 'Sans-Serif';
 font-size: 20px;    
 background-color: rgb(209, 255, 117);;
border-radius:20px;
font-color:white;


</string>
     </property>
     <widget class="QPushButton" name="pushButton_34">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#d1d938;&quot;&gt;employees&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
color:rgb(33, 33, 41);
font-family: 'Sans-Serif';
 font-size: 24px;    
 background-color: transparent;
border-radius:20px;
font-color:white;


}</string>
      </property>
      <property name="text">
       <string>code recupuration:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cin_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    font-family: 'Sans-Serif';
    font-size: 20px;
    background-color: white;  /* Couleur de fond blanche */
    border-radius: 20px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>100</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>185</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Retoure</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLineEdit" name="pass">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    font-family: 'Sans-Serif';
    font-size: 20px;
    background-color: white;  /* Couleur de fond blanche */
    border-radius: 20px;
}
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Normal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_35">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#d1d938;&quot;&gt;employees&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">
color:rgb(33, 33, 41);
font-family: 'Sans-Serif';
 font-size: 20px;    
 background-color: rgb(209, 255, 117);;
border-radius:20px;
font-color:white;


</string>
     </property>
     <property name="text">
      <string>Retape Password :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passretape">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>220</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {

    font-family: 'Sans-Serif';
    font-size: 20px;
    background-color: white;  /* Couleur de fond blanche */
    border-radius: 20px;
}
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Normal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_38">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#d1d938;&quot;&gt;employees&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">
color:rgb(33, 33, 41);
font-family: 'Sans-Serif';
 font-size: 20px;    
 background-color: rgb(209, 255, 117);;
border-radius:20px;
font-color:white;


</string>
     </property>
     <property name="text">
      <string>Nouvaux password :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>300</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color:rgb(33, 33, 41);
font-family: 'Sans-Serif';
 font-size: 20px;    
 background-color: rgb(209, 255, 117);;
border-radius:20px;
font-color:white;


}</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>185</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Retoure</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
